### PR TITLE
Build script changes for Jdk 11

### PR DESCRIPTION
**Pull request by chonton**
### Type of change
- Enhancement / new feature

### Description

Support Java11 for building the operators

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Update/write design documentation in `./design`
- [ ] Write tests
- [ ] Make sure all tests pass
- [ ] Update documentation
- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [ ] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Update CHANGELOG.md

### CODE REVIEW

**Comment by chonton** [comment]
This branch needs [OrderedProperties](https://github.com/strimzi/strimzi-kafka-operator/pull/1275) to pass Travis.

**Comment by chonton** [comment]
Ok to merge?

File: .travis.yml [hunk 1]
```diff
@@ -5,6 +5,7 @@ notifications:
     secure: c8TGIgFUIwQm/b4LKRjyqs+Lw4QCg2nd6lnwxjwsJKCJBUmJp5Pf+h8rznXJUglxytYWoLZQ7NFuFyt85L1RQQxK+rWeRn6mddEr5gR3vXJUvYycPAIWDGfVVkSsKdPVyxgbmlrSxbThB4lMHIeUUzE4ITTb9IyXxPcvTHSBPQVpED/MlJgYLucRKHpUbvNn8Bj+bKECDzbNoZTLc7gXDPIi3ejazLC2BYzxHfk52b7O18LhLMWVra8fccxlr1sBzprrzTKj10yDXrzuyv8YOcFTs/ru8wZyz+8hPk97KBk1wyEN7mFhFVPlTQP/cOMYUMM/iflWLV2iR0gmg7Y5LrBauR0CkLW30KOLR9p4B4Ygp5CuJPxrniYfKUXlB1KyUPLlDjMc8/CZ8Aunc4SniZqDg8Ow4vix/3PebGj+2rwjDL7KizMzxNSQa0ieMEUxgiWnPl5ZY8zXUkklHH+x7sB9aX3UpElnfREbq9d9w2Ak6EwvANTds4lPScieOY1gntvTCfcEethR3mMEY9vT4R9+ZXF54RKogbBFG89k+1J2Dv0rI8ZZTpwYIkNWHpxfIeKz4jekOm0lFJE/8/zWIgyZEZSqIi6GhfiFaEUzOKXG295VfDhE5KF+uzXVHeWPA8Hgy1aggCtXc/hBam3L8Ec2gaSEU547MNuLOMQkEhY=
 jdk:
 - oraclejdk8
+- openjdk11
```
Comment (chonton) [inline .travis.yml]: Maybe do not want to commit this?

Comment (scholzj) [reply .travis.yml]: I think we should try to keep it. Otherwise the Java 11 support might break sooner or later without us noticing. WDYT @tombentley?

Comment (tombentley) [reply .travis.yml]: I have no objection to keeping it.

**Comment by scholzj** [comment]
Thanks for the PR.

**Pull request merged**